components/solr-access-service/api/src/main/resources/ApplicationResources.properties

```diff
@@ -1,2 +0,0 @@
-phenotips.solrAccessService.noMatches=No matches found
```

Comment:
These translations are still used, perhaps they should be removed in #2293 instead.